src/index.ts

```diff
@@ -78,6 +78,61 @@ if (stylesheet) {
   fs.copyFileSync('src/styles/index.css', `${output}/index.css`);
 }
 
+/**
+ * Process markdown for H1 (e.g: # headhing 1)
+ * @param data Data to be processed
+ * @return return array where h1 markdown is processed
+ */
+const processH1Markdown = (data: string[]): string[] => {
+  return data.map((text) => {
+    if (text.indexOf('# ') !== -1) {
+      if (text.substr(0, 2) === '# ') return `<h1>${text.substr(2)}</h1>`;
+    }
+    return `${text}`;
+  });
+};
+
+/**
+ * Process markdown for H2 (e.g: ## headhing 2)
+ * @param data Data to be processed
+ * @return return array where h2 markdown is processed
+ */
+const processH2Markdown = (data: string[]): string[] => {
+  return data.map((text) => {
+    if (text.indexOf('## ') !== -1) {
+      if (text.substr(0, 3) === '## ') return `<h2>${text.substr(3)}</h2>`;
+    }
+    return `${text}`;
+  });
+};
+
+/**
+ * Process markdown for P
+ * @param data Data to be processed
+ * @return return array where p markdown is processed
+ */
+const processPMarkdown = (data: string[]): string[] => {
+  return data.map((text) => {
+    if (text.substr(0, 1) !== '<' && text.substr(text.length - 1, 1) !== '>')
```

Comment:
What's the purpose of checking if `text` starts with `<` ends with `>`?

Reply:
Small tip, `text.substr(text.length - 1, 1)` is the same as `text.substr(text.length - 1)` ( without the second argument).

Reply:
This is checking if it is wrapped to something like h1. If it is, do not need to turn into paragraph